src/Kerbero.Common/Entities/INukiOptions.cs

```diff
@@ -0,0 +1,6 @@
+namespace Kerbero.Common.Entities;
+
+public interface INukiOptions
```

Comment:
why?

Reply:
He needs info from the appsettings (a lot)